### PR TITLE
Portfolio project: Mobile first and linters checked

### PR DESCRIPTION
**Pull request by Booze33**
Features
-Responsive for mobile screen
-Logo and menu icon created
-Flexible colors

Bugs fixed
-Linters errors fixed
-CSS responsive fixed
-HTML errors fixed

Files
-index.html
-style.css
-Resources file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Chivo+Mono:wght@100&family=Noto+Sans+JP&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="header">
+    <header>My Logo</header>
+    <div class="menu-img"><img src="images/Union.svg"></div>
+    </div>
+    <section id="first-sec">
+        <h1>I'm Tebe <br> Glad to see <br> you!</h1>
```
Comment (bobb-Rob) [inline index.html:19]: - [ ] The Font-family used here is at variance from what is expected which is "Poppins". Please ensure that you use the correct font type across this project with the same provided in the Figma design. 

Your design | Figma Design
---- | ----
image | image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <link rel="stylesheet" href="style.css"> 
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Chivo+Mono:wght@100&family=Noto+Sans+JP&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="header">
+    <header>My Logo</header>
+    <div class="menu-img"><img src="images/Union.svg"></div>
+    </div>
```
Comment (bobb-Rob) [inline index.html:17]: - [ ] The container of this section should be a `header` tag. * Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. Please change this `<div class="header">` to a `<header>` tag.
- [ ] The logo should be wrapped with a link (`a` tag). This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. Kindly replace the current header tag on the logo with an `a` tag.
- [ ] The hamburger button should be wrapped in a `nav` tag. * The `nav` tag is for semantics reasons, and the right position for convention and user experience for mobile devices.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <section id="first-sec">
```
Comment (bobb-Rob) [inline index.html:18]: - [ ] It is a core requirement that the content within this section is aligned with `flexbox`. Please ensure that you have used flexbox.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tonic">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+        </ul>
+        <button>See Project</button>
+    </section>
```
Comment (bobb-Rob) [inline index.html:47]: - [ ] Kindly remove this section as it is not part of what is required for this milestone.

File: README.md [hunk 5]
```diff
@@ -1 +1,225 @@
...
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello Microverse!]** is a project for week 1.
```
Comment (bobb-Rob) [inline README.md:42]: - [ ] Please update the project title here with the correct one.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Booze33 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @Booze33, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] ✅ No Linter error
- [ ] ✅ Correct use of GitHub flow
- [ ] ✅ Very descriptive Pull Request(PR)
- [ ] ✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**